mosaic-site/src/components/components/ComponentsList.tsx

```diff
@@ -0,0 +1,236 @@
+  const { devStatus, name, storybookUrl } = component;
+
+  return devStatus === ComponentStatus.READY && storybookUrl ? (
+    <Link href={storybookUrl}>
+      <span>{name}</span>
+    </Link>
+  ) : (
+    <span>{name}</span>
+  );
+};
+
+const ComponentStatusData = ({
+  status,
+  availableSince,
+}: {
+  status: ComponentStatus;
+  availableSince?: string;
+}) => {
+  const showReleaseDate = availableSince && status === ComponentStatus.READY;
+  const isMobileView = useIsMobileView();
+  const mobileView = (
+    <span>{showReleaseDate ? `v${availableSince}` : null}</span>
+  );
+
+  const activeStatus = status !== ComponentStatus.NOT_APPLICABLE;
+
+  return (
+    <div className={clsx(styles.status, styles[statusClass(status)])}>
+      {activeStatus ? <StepActiveIcon /> : null}
+      {isMobileView && activeStatus ? (
+        mobileView
+      ) : (
+        <span>
+          {showReleaseDate ? `Released in v${availableSince}` : status}
+        </span>
+      )}
+    </div>
+  );
+};
+
+type ComponentHeaderProps = {
+  logo: JSX.Element;
+  label: string;
+  isSorted: boolean;
+  ascendingOrder: boolean;
+  handleClick: () => void;
+};
+
+const ComponentHeader = ({
+  logo,
+  label,
+  isSorted,
+  ascendingOrder,
+  handleClick,
+}: ComponentHeaderProps) => {
+  const isMobileView = useIsMobileView();
+  const arrowIcon = ascendingOrder ? <ArrowUpIcon /> : <ArrowDownIcon />;
+  return (
+    <Button onClick={handleClick}>
+      <span className={styles.headerContainer}>
+        <span>
+          {logo}
+          {!isMobileView && <span>{label}</span>}
+        </span>
+        {isSorted && arrowIcon}
+      </span>
+    </Button>
+  );
+};
+
+export const ComponentsList = () => {
+  const [componentsList, setComponentsList] = useState(
+    componentsListSorting("devStatus", true)
+  );
+
+  const [isSortedBy, setIsSortedBy] = useState<SortedBy>("devStatus");
+
+  const [hasAscendingOrder, setHasAscendingOrder] = useState(true);
+
+  const handleSorting = (componentDetail: SortedBy) => {
+    setIsSortedBy(componentDetail);
+
+    if (componentDetail === isSortedBy) {
+      setHasAscendingOrder((hasAscendingOrder) => !hasAscendingOrder);
+      setComponentsList(
+        componentsListSorting(componentDetail, !hasAscendingOrder)
+      );
+    } else {
+      // we want to reset the sorting order when switching to a different column
+      setHasAscendingOrder(true);
+      setComponentsList(componentsListSorting(componentDetail, true));
+    }
+  };
+
+  const ariaSort = hasAscendingOrder ? "ascending" : "descending";
+
+  return (
+    <div className={styles.componentList}>
+      <Table>
+        <thead>
+          <tr>
+            <th aria-sort={isSortedBy === "name" ? ariaSort : "none"}>
+              <ComponentHeader
+                logo={
+                  <Image
+                    src="/img/storybook_logo.svg"
+                    alt="storybook logo"
+                    nextImageClassName={""}
```

Comment:
just wondering what this is, and if it should be an empty str?

Reply:
This should probably be an optional prop, i raised a PR with a fix https://github.com/jpmorganchase/mosaic/pull/242

Reply:
> This should probably be an optional prop, i raised a PR with a fix [jpmorganchase/mosaic#242](https://github.com/jpmorganchase/mosaic/pull/242)

that's perfect, thank you!

Reply:
this has been merged :-)

Reply:
Thank you! @Fercas123 @libertymayc are you ok for me to merge this in and I'll raise a separate PR once there's been a release?

Reply:
sure!